assets/js/util.js

```diff
@@ -293,15 +298,74 @@ function callApy(options, endpoint) {
     var requestUrl = window.location.protocol + window.location.hostname +
         window.location.pathname + '?' + $.param(requestOptions.data);
 
+    callApyStartTime = Date.now();
     if(requestUrl.length > THRESHOLD_REQUEST_URL_LENGTH) {
         requestOptions.type = 'POST';
         return $.ajax(requestOptions);
     }
     return $.jsonp(requestOptions);
 }
 
+function callApyDuration() {
+    callApyEndTime = Date.now();
+    checkServiceLoadTimes(callApyEndTime - callApyStartTime);
+}
+
+function checkServiceLoadTimes(requestDuration) {
+    var individualDurationThreshold = 2000;
+    var cumulativeDurationThreshold = 1500;
+    var demoThreshold = 10000; // for demo purposes only
+    if(store.able()) {
+        if(sessionStorage.requestsMade) {
```

Comment:
`store.able()` doesn't check if the browser has `window.sessionStorage`, it checks if you can use the other `store.js` functions (whose implementation the caller shouldn't need to know about).

So there's 

1. the [silently-failing](https://stackoverflow.com/a/12337672/69663) solution  (which doesn't seem too bad in this case), or
2. something simple like `if(window.sessionStorage)` or
3. the more thorough `if(window.sessionStorage && (window.sessionStorage.test="a") === window.sessionStorage.test && window.sessionStorage.test === "a")`.